src/router/db.ts

```diff
@@ -95,3 +95,26 @@ db.post('/sync', async (req, res) => {
     res.status(500).end(logger.error(e));
   }
 });
+
+db.get('/get/:dbAlias', async (req, res) => {
+  const { dbAlias, } = req.params;
+  console.log('dbalias', dbAlias)
+  if (!dbAlias) return res.status(400).json("Required param 'dbAlias' not provided");
+  try {
+    const dbs = await iasql.list(dbMan.getUid(req.user), dbMan.getEmail(req.user), false);
```

Comment:
you can just call the `getDb` method from the respository directory since here you are fetching all of the rows?